tardis/montecarlo/montecarlo_numba/r_packet.py

```diff
@@ -1,6 +1,8 @@
 from enum import IntEnum
+from operator import index
```

Comment:
not sure where this is coming from.

Reply:
Even I am not sure where did it came from, like I did not add it myself. But its not there now. Also, I have rebased it.